components/cloud/aws/src/kms.rs

```diff
@@ -0,0 +1,278 @@
+use std::future::Future;
```

Comment:
This code is taken unchanged from the encryption kms.rs file. https://github.com/tikv/tikv/pull/9576/files#diff-0f18341af3dc671328d8f26275bacc9676378054ce65b6eba9e561d69511b4c9